src/main/java/be/seeseemelk/mockbukkit/inventory/meta/ItemMetaMock.java

```diff
@@ -47,7 +47,8 @@ public ItemMetaMock(ItemMeta meta) {
         unbreakable = meta.isUnbreakable();
         enchants = new HashMap<>(meta.getEnchants());
         customModelData = meta.hasCustomModelData() ? meta.getCustomModelData() : null;
-        
+        hideFlags.addAll(meta.getItemFlags());
```

Comment:
"itemFlags" may be more appropriate for this variable, just noticing.

Reply:
I noticed that too, but didn't mention it because it wasn't added here